src/pages/home/HeaderView.js

```diff
@@ -176,12 +178,29 @@ const HeaderView = (props) => {
                                     shouldUseFullTitle={isChatRoom || isPolicyExpenseChat || isThread || isTaskReport}
                                 />
                                 {(isChatRoom || isPolicyExpenseChat || isThread) && !_.isEmpty(subtitle) && (
-                                    <Text
-                                        style={[styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting, styles.pre]}
-                                        numberOfLines={1}
-                                    >
-                                        {subtitle}
-                                    </Text>
+                                    <>
+                                        {isThread ? (
+                                            <PressableWithoutFeedback
```

Comment:
@chiragsalian, We are getting console type error 

"Failed prop type: Provide a valid string for accessibilityLabel prop when accessible is true"

By default `accessible` is true in case of `PressableWithoutFeedback`, 
could you either give `accessibilityLabel` or set `accessible` false if we don't want it to be accessible to unblock us?

Reply:
oh i missed this, looking now 👀

Reply:
off the top of your head do you know whats `accessible`? I'm wondering if setting `accessibilityLabel` or `accessible` is better for subheaders.

Reply:
nvm, i found it.